### DISCUSSION TITLE
How to get vim keybinds inside cmd?

### DISCUSSION THREAD

**Discussion by prescientmoon** [folke/noice.nvim#205] [Q&A] [answered]
The CMD seems to be stuck in insert mode right now. Getting vim keybinds inside was the main reason I installed this plugin. Is there a way to get this to work?

Thanks in advance!

**Comment by folke** [comment, marked as answer]
The Noice cmdline **IS** the cmdline. It just looks different, but behaves exactly the same.

Any `cmap` keybindings should work as before.

**Reply by prescientmoon** [reply]
@folke I want to be able to do stuff like enter normal mode and whatnot. Is there no way to achieve this using this plugin?

**Reply by folke** [reply]
As I said, Noice is an external ui on the cmdline. What you see is different, but how it works is **exactly** the same as before. So no, you can not enter normal mode with Noice.

**Reply by prescientmoon** [reply]
Ah, what a shame:( Guess I'll have to look for some other plugin
